src/constants/labels.json

```diff
@@ -30,9 +30,9 @@
       "newTry": "Neuer Versuch",
       "feedback": {
         "articleMissing": "Ups, deine Eingabe ist nicht vollständig. Füge einen Artikel ein.",
-        "wrong": "Die richtige Antwort ist: ",
-        "almostCorrect1": "Deine Eingabe ",
-        "almostCorrect2": " ist fast richtig.",
+        "wrong": "Die richtige Antwort ist:",
+        "almostCorrect1": "Deine Eingabe",
+        "almostCorrect2": "ist fast richtig.",
```

Comment:
Sieht das nicht doof aus, wenn die Leerzeichen fehlen?

Reply:
There were double spaces, one from the labels here and one inserted in the jsx code. I think it is better to just always have labels with no padding spaces, that way you can use it whatever way you want and can easily insert spaces if needed.